server/models/Profile.js

```diff
@@ -0,0 +1,43 @@
+const mongoose = require("mongoose");
+const userSchema = require("./User");
+const email = User.find_by_id({});
+const profileSchema = new mongoose.Schema({
+  firstName: {
+    type: String,
+    required: true,
+  },
+  lastName: {
+    type: String,
+    required: true,
+  },
+  dob: {
+    type: Date,
+    required: true,
+  },
+  user: {
+    type: userSchema,
+    required: true,
+  },
+  userPhotoUrl: {
+    type: String,
+  },
+  description: {
+    type: String,
+  },
+  availability: {
+    type: String,
```

Comment:
How are we planning to store availability here? If we are using 'String', won't this be an array of strings here?

Reply:
Yes it would be an array, Thanks of catching that. Fixed.